Functions-request-config.js

```diff
@@ -26,7 +27,7 @@ const requestConfig = {
   // location of source code (only Inline is curently supported)
   codeLocation: Location.Inline,
   // location of secrets (only Inline is currently supported)
-  secretsLocation: Location.Inline,
+  secretsLocation: Location.Offchain,
```

Comment:
We don't want this to be a default for our example request. The example should work out of the box.